tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php

```diff
@@ -26,6 +26,9 @@ public function setUp() {
 		$this->module = new DeletePostsByTaxonomyModule();
 	}
 
+	/**
+	 * Test deleting posts from a single taxonomy term default post type.
+	 */
 	public function test_deleting_posts_from_built_in_taxonomy_terms() {
```

Comment:
@rajanit2000 - The function name says that posts are `deleted`. But looking at the `force_delete` filter, you're only `trashing` and **not** `deleting`. Kindly **change** the function name respectively.

Check the following line - https://github.com/sudar/bulk-delete/pull/382/files#diff-de80f6e764d3cf5960c34228622a60d4R50